build.gradle

```diff
@@ -39,22 +39,12 @@ run {
     confFile = "file:" + File.separator + File.separator + "$projectDir/conf/sidecar.yaml"
     jvmArgs = ["-Dsidecar.logdir=./logs",
                                  "-Dsidecar.config=" + confFile,
+                                 "-Dlogback.configurationFile=./conf/logback.xml",
                                  "-Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.SLF4JLogDelegateFactory",
-                                 "-javaagent:$projectDir/src/dist/agents/jolokia-jvm-1.6.0-agent.jar=port=7777,host=localhost"]
+                                 "-javaagent:$projectDir/agents/jolokia-jvm-1.6.0-agent.jar=port=7777,host=localhost"]
 }
 
 sourceSets {
-    // This is needed as gradle considers `src/main/resources` as the default resources folder
-    main {
-        resources {
-            srcDirs = ['conf', 'setup', 'src/main/resources']
```

Comment:
I believe it's no longer needed as I relocated `conf` to `src/main/dist/conf` so it gets pulled within the distribution.  `setup` directory doesn't appear to exist, and `src/main/resources` is implicitly a resource directory so it doesn't need to be defined.

The removed `test { resources` step isn't necessary now since I added a test specific test-logback.xml

Reply:
👍